lib/components/ChangelogContainer/index.tsx

```diff
@@ -29,14 +52,18 @@ export const ChangelogContainer: React.FC<Props> = ({
     },
   });
 
-  if (error) {
-    console.error(error);
+  if (errorMessage) {
+    console.error(errorMessage);
   }
 
   return (
     <div>
-      <ChangelogContext.Provider value={{ loading, error, data }}>
-        {children}
+      <ChangelogContext.Provider value={{ data }}>
```

Comment:
Ich würde den outer div entfernen. So forced to bei Benutzung eine Mutation im DOM